saleor/payment/utils.py

```diff
@@ -693,11 +693,37 @@ def payment_owned_by_user(payment_pk: int, user) -> bool:
     )
 
 
+def get_correct_event_types_based_on_request_type(request_type: str) -> List[str]:
```

Comment:
```suggestion
def get_correct_event_types_based_on_request_type(request_type: str) -> list[str]:
```
From python 3.9 onward using `list` over `typing.List` is preferred way.